exchange-client.gemspec

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'openstax/exchange/client/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "openstax-exchange-client"
```

Comment:
I have no strong preference personally. You can find gems that interface with some API named both ways:

https://github.com/google/google-api-ruby-client

https://github.com/lemurheavy/coveralls-ruby